packages/openapi-v3-types/CHANGELOG.md

```diff
@@ -0,0 +1,138 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="4.0.0-alpha.25"></a>
```

Comment:
Since this is a new package, it should start with empty (or no) CHANGELOG.

Reply:
good catch 👍